src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public void getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Ball ball = new Ball();
+        ball.setColor("" + colorSupplier.getRandomColor());
+        Random random = new Random();
+        ball.setNumber(random.nextInt(100));
+        System.out.println(ball);
```

Comment:
Don't create `ColorSupplier` and `Random` here, instead move it to the class-level.

Also, don't use `"" + colorSupplier.getRandomColor()`, instead call method `name()`. [Reference](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants).

And don't. use magic number `100`.